examples/dagster_feray_example/defs/features/feature_example.py

```diff
@@ -0,0 +1,73 @@
+from typing import Any
+
+import polars as pl
+from dagster import Config, job, op
+from dagster_feray.resources import build_feature_assets
+from feray.data_structures import FeatureDefinition
+from feray.service import FeatureRegistry
+
+users_def = FeatureDefinition(
+    asset_key="users",
+    compute_fn=lambda _: pl.DataFrame({"user_id": [1, 2, 3]}),
+    output_features=["user_id"],
+    entity_keys=["user_id"],
+)
+
+locations_def = FeatureDefinition(
+    asset_key="locations",
+    compute_fn=lambda up: up["users"].with_columns(country=pl.lit("USA")),
```

Comment:
Would it be helpful for the sake of beginner-friendly examples to have one of these FeatureDefinition examples reference an actual function object? It could then be better type-hinted and docstring'd so make the example super clear for interested new users.

Reply:
oh definitively - this was just really a mini showcase and not a full blown example. The idea is to finaliize

- the API/protocol
- the implementation

and then add some more full blown examples. I think this is critical for adoption of the package